communitiesuk/funding-service-pre-award-stores
# FSPT-54: Make Fund Store work in new repo

### Change description
Lift & shift of `fund-store` into `pre-award-stores` and basic tweaks to make this work as a single application with `fund_store` specific code in its own module.

All existing `fund-store` tests pass.



### How to test
- Check out `feature/FSPT-56-add-combined-stores-repo` on the [docker-runner repo](https://github.com/communitiesuk/funding-service-design-docker-runner/)
- Do `make down` and then `make up` to build the `pre-award-stores` service and the `pre_award_stores` database
- Open apply/assess front-ends locally and check the fund data is pulling through correctly
- Hit [http://localhost:3012/healthcheck](http://localhost:3012/healthcheck), [http://localhost:3012/ui](http://localhost:3012/ui) and [http://localhost:3012/fund/ui](http://localhost:3012/fund/ui) in your browser and test the endpoints
- Run pytest and all root tests and `fund_store` tests should pass.
